### ISSUE TITLE
Allow to customize the key used by the middlewares in the request attributes

### ISSUE THREAD

**Issue by oscarotero** [oscarotero/psr7-middlewares#1]
For example:

``` php
use Psr7Middlewares\Middleware\AuraSession;

AuraSession::setKey('_AppSession');
```

This allows to prevent conflicts and use static functions to get the values:

``` php
use Psr7Middlewares\Middleware\AuraSession;

function ($request, $response, $next) {
    $session = $request->getAttribute('_AppSession');
    //this is the same than:
    $session = AuraSession::getSession($request);
}
```

**Comment by ojhaujjwal** [comment]
:+1:

**Comment by oscarotero** [comment]
Fixed in [3.0](https://github.com/oscarotero/psr7-middlewares/releases/tag/v3.0.0).
Instead customize the key used by each middleware, now all values are stored under the "Psr7Middlewares\Middleware" key (using an array as value).
